station/micropython/apps/temperature_tft.py

```diff
@@ -5,7 +5,7 @@
 
 import vga1_16x32 as font
 
-pin_temp = ADC(Pin(12))
+pin_temp = ADC(Pin(36))
```

Comment:
Pin 36 is on ADC1 and works with WIFI. Pin 12 was on ADC2 and does not work with WIFI.